data-pipeline/src/data_pipeline/datasets/clinvar.py

```diff
@@ -162,34 +162,30 @@ def _parse_variant(variant_element):
     return variant
 
 
-def import_clinvar_xml(clinvar_xml_path):
-    release_date = None
-
-    clinvar_xml_local_path = os.path.join("/tmp", os.path.basename(clinvar_xml_path))
-    print("Copying ClinVar XML")
-    if not os.path.exists(clinvar_xml_local_path):
-        subprocess.check_call(["gsutil", "cp", clinvar_xml_path, clinvar_xml_local_path])
+def parse_clinvar_xml_to_tsv(
```

Comment:
nice little refactor here, also a good demonstration of one reason significant indentation is a bad idea: it makes it much more likely to introduce an error in this kind of refactoring by moving a block of code but neglecting to adjust indentation